lib/api/hub_config_api.rb

```diff
@@ -1,20 +1,17 @@
 class HubConfigApi
+  include HubEnvironmentConcern
   require 'cgi'
-
-  HUB_HOST = Rails.configuration.hub_config_host
   HEALTHCHECK_ENDPOINT = 'service-status'.freeze
   CERTIFICATES_ROUTE = '/config/certificates/'.freeze
   CERTIFICATE_ENCRYPTION_ENDPOINT = "%{entity_id}/certs/encryption".freeze
   CERTIFICATES_SIGNING_ENDPOINT = "%{entity_id}/certs/signing".freeze
 
-  def initialize; end
-
-  def healthcheck
-    Faraday.get healthcheck_path
+  def healthcheck(environment)
+    build_request(**healthcheck_path(environment))
```

Comment:
I'm not a biggest fan of the double splat (`**`), in my opinion it makes stuff less readable, but I can see the reasons here...

Reply:
it has its place :)

Reply:
Initially I was doing this 
```
build_request(healthcheck_path(environment), environment)
```
then this
```
health_check_path = healthcheck_path(environment)
build_request(healthcheck_path, environment)
```
I didn't like have to use a variable for each of the sections. so ended up with the current implementation